nixnet_examples/can_signal_single_point_io.py

```diff
@@ -62,14 +74,14 @@ def main():
                 num_signals = len(value_buffer)
                 signals = input_session.read_signal_single_point(num_signals)
                 for timestamp, value in signals:
-                    date = datetime.datetime.fromtimestamp(timestamp / 1e9)
+                    date = datetime.datetime.fromtimestamp(timestamp / 1e9) + delta
```

Comment:
I assume this is still not working.

its fine if it doesn't, I just wanted to confirm